Changelog.md

```diff
@@ -225,6 +226,27 @@ The new balance pack manager version is `1.3`.
 - Added `inaccuracy`
 - Removed `headshotMultiplier`
 
+### New Guns Renewed and Added
+
+//  Renewed Guns
+
+- M200 Intervention
+- M79(WIP)
+- Colt M1911
+- M60E4(WIP)
+- Saiga12 (WIP)
+- Uzi
+- APC9
+- S7-10 Tricun
+- AS50
+- AUG(WIP)
+
+//  New Guns
+
+- Accuracy International AX50
+- Modernized Kalashnikov Machine Gun(PKM)
+- PGM Hécate II(WIP)
+
```

Comment:
No, this either goes in added or changed

Reply:
That's all the changes that are added, the reason why I put (WIP) is because there are missing animations